OWNERS_ALIASES

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
+
+aliases:
```

Comment:
Can we use the github team `@rhobs/platform-sre` , or does the CI tool not know about teams?

Reply:
Unfortunately GH teams are not supported